### PR TITLE
Use immediate = TRUE

### PR DESCRIPTION
**Pull request by krlmlr**
for statements. Important on SQL Server, should not hurt other databases.

### CODE REVIEW

**Comment by krlmlr** [comment]
All tests with MSSQL succeed locally when I add the test source. 

Tests fail without development version of {odbc}. For FreeTDS we also need https://github.com/r-dbi/odbc/pull/360.

**Comment by krlmlr** [comment]
Tests are now run on GitHub Actions too.

File: DESCRIPTION [hunk 1]
```diff
@@ -41,13 +41,16 @@ Suggests:
...
     nycflights13,
+    odbc (>= 1.2.2.9000),
     RMariaDB (>= 1.0.2),
     rmarkdown,
     RPostgres (>= 1.1.3),
     RSQLite (>= 2.1.0),
     testthat (>= 2.0.0)
 VignetteBuilder: 
     knitr
+Remotes:
```
Comment (hadley) [inline DESCRIPTION:52]: Can you not use remotes, and skip if the version isn't high enough? That way I don't have to worry about updating it when I release to CRAN?

**Comment by krlmlr** [comment]
I'll also add explicit mssql tests.

**Pull request merged**